src/lib.rs

```diff
@@ -515,7 +561,7 @@ impl SharedChild {
 }
 
 /// A list of commands to run.
-#[derive(Debug, Clone)]
+#[derive(Clone, Debug, Default)]
```

Comment:
I assume you added this Default because something somewhere forced you to?

This is still an API change as you are adding stuff to the API of CommandList. If you can avoid it, it's best.

In this case it makes particularly no sense to have it.

Reply:
Nice one, I forgot to remove it.